GiniVision/Classes/Core/CameraViewController.swift

```diff
@@ -279,7 +279,7 @@ public typealias CameraScreenFailureBlock = (_ error: GiniVisionError) -> Void
     public override func loadView() {
         super.loadView()
         edgesForExtendedLayout = []
-        view.backgroundColor = .black
+        view.backgroundColor = GiniConfiguration.shared.backgroundColor
```

Comment:
Here, it gets even worse. The configuration **is** injected, but here you use the shared instance, This can lead to subtle bugs.